content/en/docs/contrib/writing-code.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Writing code
+description: Fix a bug, or add a new feature. You can make a pull request and see your code in the next version of SpinKube!
+weight: 10
+---
+
+Interested in giving back to the community a little? Maybe you've found a bug in SpinKube that you'd
+like to see fixed, or maybe there's a small feature you want added.
+
+Contributing back to SpinKube itself is the best way to see your own concerns addressed. This may
+seem daunting at first, but it's a well-traveled path with documentation, tooling, and a community
+to support you. We'll walk you through the entire process, so you can learn by example.
+
+## Who's this tutorial for?
+
+For this tutorial, we expect that you have at least a basic understanding of how SpinKube works.
+This means you should be comfortable going through the existing tutorials on deploying your first
+app to SpinKube. It is also worthwhile learning a bit of Rust, since many of SpinKube's projects are
+written in Rust. If you don't, [Learn Rust](https://www.rust-lang.org/learn) is a great place to
+start.
+
+Those of you who are unfamiliar with `git` and GitHub will find that this tutorial and its links
+include just enough information to get started. However, you'll probably want to read some more
+about these different tools if you plan on contributing to SpinKube regularly.
+
+For the most part though, this tutorial tries to explain as much as possible, so that it can be of
+use to the widest audience.
+
+## Code of Conduct
+
+As a contributor, you can help us keep the SpinKube community open and inclusive. Please read and
+follow our [Code of Conduct](https://www.fermyon.com/code-of-conduct).
```

Comment:
Do we or will we have an equivalent CoC under the SpinKube org?  (Follow-up?)

Reply:
Oh! Apparently there is one now via https://github.com/spinkube/governance/blob/main/CODE_OF_CONDUCT.md.

I copied this from somewhere else in the spinkube org... I'll have to track that down and fix it.

Good catch.

Reply:
filed https://github.com/spinkube/spin-operator/pull/288 to fix the link in spin-operator.